recipes/newrelic/apm/php-agent/php.yml

```diff
@@ -110,12 +190,12 @@ install:
           # Web server was detected. Send transaction associated with the webserver php.ini
           # and the newrelic.ini file associated with that web server.
           #
-             php -c "${WEB_PHP_INI_DIR}/php.ini" -c "${WEB_NR_INI_DIR}/newrelic.ini" -n --ini &>/dev/null
-             php -c "${WEB_PHP_INI_DIR}/php.ini" -c "${WEB_NR_INI_DIR}/newrelic.ini" -n --ini &>/dev/null
-             php -c "${WEB_PHP_INI_DIR}/php.ini" -c "${WEB_NR_INI_DIR}/newrelic.ini" -n --ini &>/dev/null
+             sudo -u ${nonpriv_user} php -c "${WEB_PHP_INI_DIR}/php.ini" -c "${WEB_NR_INI_DIR}/newrelic.ini" -n --ini &>/dev/null
+             sudo -u ${nonpriv_user} php -c "${WEB_PHP_INI_DIR}/php.ini" -c "${WEB_NR_INI_DIR}/newrelic.ini" -n --ini &>/dev/null
+             sudo -u ${nonpriv_user} php -c "${WEB_PHP_INI_DIR}/php.ini" -c "${WEB_NR_INI_DIR}/newrelic.ini" -n --ini &>/dev/null
           else
           #
-          # No web server detected, this was a pure PHP CLI php agent installation.
+          # No web server detected, pure PHP CLI.
```

Comment:
In this scenario, we also risk a permissions problem if we do not know which user runs the PHP CLI app.

Reply:
Possibly, but with the CLI this seems unlikely as they should be logged on as the user that will use the app(otherwise, if they aren't, there's no way for us to guess this).  If they were installing manually, they'd do these same steps.
They would download the agent, untar, run install, then run `php --info` (or another php cli command) from command prompt to verify.

Reply:
Aren't all recipes run as root?

Reply:
Recipes run as root by default. Gabe provided this snippet for how to run a command as the user who invoked sudo to run the recipe.

https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/apm/node/linux.yml#L49

Reply:
Oh!  Thanks for pointing that out, I hadn't realized, but since it's run with obviously it would be.  Thanks for the link, I'll get that incorporated.